ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1. PropTypes is a method that allows us to check and make sure that a piece of incoming data matches with a datat type that we specify within the propTypes method. We can, for example say PropTypes.string and check to see if the incoming information is a string, and if not we will recieve an error message.
+2.Lifecycle events are methods that happen throughout "life events" of a component. Methods such as render and componentDidMount can happen in these life events, which include the mounting(birth), updating(life), and unmounting(death) stages of our component.
+3.A higher order component is a component that takes in another component(s) as an argument and spits out a new component. It is reausable and allows us to add common elements such as state-fulness to multiple components, all while keeping our code DRY.
```

Comment:
Generally, an HOC is technically a function that returns a component, but this is a good answer nonetheless